.travis.yml

```diff
@@ -1,10 +1,35 @@
 sudo: false
 dist: trusty
 language: node_js
+
 node_js:
-  - '6'
   - '8'
-  - '10'
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - docker-ce
+
+branches:
+  only:
+    - master
+    # Allow builds for version tags (e.g. v1.0.0)
+    - /^v(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$/
+
+before_install:
+  # Login to NPM registry
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > .npmrc
```

Comment:
This shouldn't be necessary

Reply:
We need this for publishing npm package.